docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -13,31 +13,26 @@
 
 If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+1. Determine your operating system and it's version: 
+   ```sh 
+   cat /etc/os-release
    ```
 
-2. Install the **Intel® Graphics Compute Runtime for OpenCL™** driver components required to use the GPU plugin and write custom layers for Intel® Integrated Graphics. The drivers are not included in the package. To install it, run this script:
-   ```sh
-   sudo -E ./install_NEO_OCL_driver.sh
-   ```
-   > **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
-   
-   The script compares the driver version on the system to the current version. If the driver version on the system is higher or equal to the current version, the script does 
-   not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
-   ![](../img/NEO_check_agreement.png) 
-
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
+2. You can install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://dgpu-docs.intel.com/installation-guides/index.html) or choose one that we used in internal tests
```

Comment:
```suggestion
2. Install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your operating system. You can find instructions in the [Installation Guides of Intel® software packages for general purpose GPU capabilities](https://dgpu-docs.intel.com/installation-guides/index.html), or use one of the ways in our internal tests below:
```

Reply:
@xu-yuan1 My English is not so good but this proposal does seem like correct re-wording. It's not about the latest driver vs other ways. It's about the latest version driver vs other versions of driver.

Reply:
> I think we can provide the link of the parent page to give some information on how to select the installation guides.

@xu-yuan1 My English is not so good but this proposal does seem like correct re-wording. It's not about the latest driver vs other ways. It's about the latest version driver vs other versions of driver.

Reply:
@evgenytalanin-intel , could you suggest which part of the sentence should be changed? this link: https://dgpu-docs.intel.com/index.html? or the last sentence "use one of the ways in our internal tests below"?
I was following the original sentence and didn't change much except the link. :)